bindings/java/src/test/java/org/openucx/jucx/UcpListenerTest.java

```diff
@@ -124,6 +124,20 @@ public void testConnectionHandler() throws Exception {
             } catch (Exception ignored) { }
         }
 
+        // Test connection handler persists
```

Comment:
BTW can we remove lines 118-124 now?

Reply:
I've tried yesterday. It still fails. Actualy this PR is stuck on `ucp_listener_destroy`:
```
>>> bt
#0  0x00007fb813765965 in pthread_cond_wait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
#1  0x00007fb66a237163 in rdma_destroy_id () from /lib64/librdmacm.so.1
#2  0x00007fb66a44ddbc in uct_rdmacm_cm_destroy_id (id=0x7fb80ee05ca0) at rdmacm_cm.c:23
#3  0x00007fb66a4509a4 in uct_rdmacm_listener_t_cleanup (self=0x7fb80edf06d0) at rdmacm_listener.c:120
#4  0x00007fb6c107daac in ucs_class_call_cleanup_chain (cls=0x7fb66a658740 <uct_rdmacm_listener_t_class>, obj=0x7fb80edf06d0, limit=-1
) at type/class.c:56
#5  0x00007fb66a450b2b in uct_rdmacm_listener_t_delete (self=0x7fb80edf06d0) at rdmacm_listener.c:147
#6  0x00007fb6c05670df in uct_listener_destroy (listener=0x7fb80edf06d0) at base/uct_cm.c:219
#7  0x00007fb6c02296fd in ucp_listener_close_uct_listeners (listener=0x7fb80ee05be0) at core/ucp_listener.c:100
#8  0x00007fb6c022a2a3 in ucp_listener_destroy (listener=0x7fb80ee05be0) at core/ucp_listener.c:290
#9  0x00007fb66b5f6d04 in Java_org_openucx_jucx_ucp_UcpListener_destroyUcpListenerNative () from /hpc/mtr_scrap/users/peterr/devel/ucx
-test3/bindings/java/src/main/native/build-java/test-classes/libjucx.so
```

https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=13290&view=logs&s=8bfbeaae-4c8e-5f12-f154-edd305817000&j=da813497-1cca-54a8-f0a9-cb3fd519bc00

Checking where it deadlocks

Reply:
> I've tried yesterday. It still fails. 

What is the symptom of the failure?

Reply:
OK, seems like working now. Maybe was other fail. Will add to this PR